src/mono/wasm/wasm.proj

```diff
@@ -32,6 +32,7 @@
     <_EmccDefaultsRspPath>$(NativeBinDir)src\emcc-default.rsp</_EmccDefaultsRspPath>
     <_EmccCompileRspPath>$(NativeBinDir)src\emcc-compile.rsp</_EmccCompileRspPath>
     <_EmccLinkRspPath>$(NativeBinDir)src\emcc-link.rsp</_EmccLinkRspPath>
+    <_EmccLinkUndefinedSymbolsFile>$(NativeBinDir)src\symbols.undefined</_EmccLinkUndefinedSymbolsFile>
```

Comment:
`$(_EmccLinkUndefinedSymbolsFile)`

Reply:
I think you are generating the file under that name ?

Reply:
Yes, I'm generating the `$(NativeBinDir)src\symbols.undefined`. The `_EmccLinkUndefinedSymbolsFile` is a private property for abstracting the actual value.

I still don't understard what you are trying to suggest.

Reply:
sorry, my bad.

Reply:
This is place where the value is declared for the first time.